subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df.c

```diff
@@ -140,6 +140,7 @@ struct lll_df_sync_cfg *lll_df_sync_cfg_alloc(struct lll_df_sync *df_cfg,
 		cpu_dmb();
 		first_latest = df_cfg->first;
 		if (first_latest != first) {
+			df_cfg->last = last;
```

Comment:
Can you add here reference to this comment: https://github.com/zephyrproject-rtos/zephyr/blob/860323feac7b92da3e25b2cb9f80731c5790ce63/subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c#L285-L296?